### PR TITLE
Fixed #4758: "Info" tooltip not visible in Japanese because of long label 

### LINKED ISSUE #4758: "Info" tooltip not visible in Japanese because of long label
Steps to reproduce:
1. Switch your phone to Japanese language, or make the label in your language very long in strings XML.
2. Pretend to upload a picture
3. At the "Media details" steps, the localization for "Step 1 of 4: Media Details" is so long that the (i) tooltip button is not visible.

Maybe overflow if too long?
Or if it is only for Japanese, the label can be made shorter like `1/4: メディア詳細`.

### PR DESCRIPTION
**Pull request by vivek-gite**
**Description (required)**

Fixes #4758 

What changes did you make and why?

Changed the Japanese translation of "Step %1$d to %2d"

Tested on:
samsung a20
API level 30

**Screenshots (for UI changes only)**
image

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,7 +11,6 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.AUTHENTICATE_ACCOUNTS" />
-    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
```
Comment (neslihanturan) [inline app/src/main/AndroidManifest.xml:14]: Seems like you accidentally worked on the same branch with your previous PR. These changes seems irrelevant with current PR and should be reverted.

**Review by neslihanturan — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -367,7 +367,7 @@
   <string name="storage_permission">ストレージの許可</string>
   <string name="write_storage_permission_rationale_for_image_share">画像の投稿には、外部ストレージへのアクセス権限を許可してください。</string>
   <string name="nearby_notification_dismiss_message">今後、画像募集中の近くの場所は表示されません。ただしこの通知機能は、いつでも「設定」画面から有効にできます。</string>
-  <string name="step_count">全部で%2$d件のうち%1$d件目の手順：%3$s</string>
```
Comment (neslihanturan) [inline app/src/main/res/values-ja/strings.xml:370]: also, we never edit string files of languages manually. We only edit the default strings file and translate updates handles the rest. So this problem should be solved without ja/strings.xlm is edited.

**Review by neslihanturan — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-ja/strings.xml (hunk 2 above)
Comment (vivek-gite) [reply app/src/main/res/values-ja/strings.xml:370]: @neslihanturan Thanks for reviewing my changes. I will resolve it ASAP.

Comment (vivek-gite) [reply app/src/main/res/values-ja/strings.xml]: @neslihanturan Please once review the changes

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -370,7 +370,7 @@
 
   <string name="write_storage_permission_rationale_for_image_share">We need your permission to access the external storage of your device in order to upload images.</string>
   <string name="nearby_notification_dismiss_message">You won\'t see the nearest place that needs pictures anymore. However, you can re-enable this notification in Settings if you wish.</string>
-  <string name="step_count">Step %1$d of %2$d: %3$s</string>
+  <string name="step_count">%1$d/%2$d: %3$s</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml:373]: Hi @vivek-gite , thanks for the changes but I think it looks irrelevant. With this change you change the string for all of the languages. What should be done is making the field ıi element visible without changing the text.

**Review by neslihanturan — CHANGES_REQUESTED** [review]

**Comment by neslihanturan** [comment]
Hi @vivek-gite , is this PR still active?
